README.md

```diff
@@ -261,6 +261,11 @@ PaStA and Patchwork
 --------------------
 The results of PaStA's analyses can be used by [Patchwork](http://jk.ozlabs.org/projects/patchwork/).
 
+Work on PaStA and Patchwork integration was also undertaken as a [Google Summer
+of Code](https://summerofcode.withgoogle.com/) project in 2020. The work done
+is described in detail in this [blog
+post](https://rsarky.github.io/2020/08/10/pasta-patchwork.html)
```

Comment:
Could you please add the raw content to PaStA? I don't want to loose those documents at some point.

Reply:
Yes, should I create a  `documentation` directory and add it there?

Reply:
Sounds good!

Reply:
done